connector/src/it/scala/com/datastax/spark/connector/cql/ContinuousPagingScannerSpec.scala

```diff
@@ -76,6 +83,40 @@ class ContinuousPagingScannerSpec extends SparkCassandraITFlatSpecBase with Defa
 
     withClue(sessions.map(_.toString).mkString("\n"))(sessions.size should be(1))
   }
+
+  private def executeContinuousPagingScan(readConf: ReadConf): Statement[_] = {
```

Comment:
Question: should we define the helper `private` method(s) at the bottom?

Reply:
I'll move it under `overwrite`s to more strictly follow scala convention.